Formula/libpeas.rb

```diff
@@ -10,18 +10,42 @@ class Libpeas < Formula
     sha256 "0f521913ca0eaf13b55aacb75e4b87a730be1f527215741ae2ba207caac523b2" => :el_capitan
   end
 
+  option "with-python", "install support for running Python plugins"
+
   depends_on "gettext" => :build
   depends_on "intltool" => :build
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "gobject-introspection"
   depends_on "gtk+3"
+  depends_on "python" => :optional
+  depends_on "python3" if build.with? "python"
```

Comment:
Not sure it wouldn't follow Homebrew's convention better to separate out the `python`s here into `python` and `python3` options. Throwing to @ilovezfs for an opinion on that.

Reply:
`libpeas` can support both, python 2 and 3 in parallel. I would also assume that most people will have both python versions installed.

Reply:
That was to say, I believe a single switch is all that's needed.

Reply:
Yeah I'd separate it out. `--with-python` implies python2 specifically in Homebrew currently.

Reply:
I did some experiments, trying to separate `python2` and `python3` into separate switches.

There are two PyGObject formulae: `pygobject` and `pygobject3`.
* `brew install pygobject` installs [version 2.28.6 from June 2011](https://pygobject.readthedocs.io/en/latest/changelog.html#id102)
* `brew install pygobject3` installs [version 3.26.1 from October 2017](https://pygobject.readthedocs.io/en/latest/changelog.html#id4) with support for `python2` only

When installing `libpeas --with-python`, and with a dependency on `pygobject`, the `configure` script of `libpeas` fails, complaining that PyGObject was not installed. So it seems the current `libpeas` no longer works with the 2011 `pygobject` version.

When installing `libpeas --with-python` only, and with a dependency on `pygobject3`, it depends on whether `pygobject3` was built `--with-python3`. (NOTE: `python2` support is always built into `pygobject3`, unless excluded with `--without-python`. END NOTE)

`libpeas` | `pygobject3` | `pygobject3 --with-python3`
------- | ------------ | ------------------------------
`--with-python` | :one: builds ok | :two: build fails 
`--with-python3` | :three: build fails  | :four: builds ok 
`--with-python --with-python3` | :five: build fails | :six: builds ok 

So there are three user stories we have to consider:
1. The user has installed `python2` only, and therefore has installed `pygobject3` without `python3` support. In this case, `libpeas --with-python` builds fine.
1. The user has installed `python3` only, and therefore has installed `pygobject3` with `python3` support, but without `python2` support. In this case, `libpeas --with-python3` builds fine.
1. The user has installed `python2` and `python3` in parallel, and therefore has installed `pygobject3` with support for both, `python2` and `python3`. In this case, all combinations will work, **except** when he tries to compile `libpeas --with-python`, i.e. with `python2` support only, despite having `python3` already installed. This corresponds to :two: in the table above.

I don't know what a motivation could be for doing user story no. 3, but then, you never know.

What could possible approaches be for making :two: work, short of fixing `pygobject3`? If we specify a `libpeas` build-only dependency on `pygobject3` sans `python3` support, it would build, but presumably fail at runtime.

The two easiest options would seem:
* leave as is now, i.e. always build support for both python versions;
* or accept :two: as a corner case that can be ignored since people doing this don't deserve any better anyway.

Looking forward to your thoughts.